Functions.Templates/Functions.Templates.csproj

```diff
@@ -296,6 +309,7 @@
   <ItemGroup>
     <Compile Include="Templates\IotHubTrigger-CSharpVS\IotHubTriggerCSharp.cs" />
   </ItemGroup>
+  <ItemGroup />

```

Comment:
nit: this is not needed

Reply:
Thanks VS, I didn't need this. (Thanks for catching this, Naren.)